boards/e180-zg120b-tb/doc.txt

```diff
@@ -0,0 +1,199 @@
+/**
+ * @defgroup    boards_e180-zg120b-tb Ebyte E180-ZG120B-TB Test Board
+ * @ingroup     boards
+ * @brief       Support for Ebyte E180-ZG120B-TB Test Board
+
+## Overview
+Ebyte E180-ZG120B Test Board is equipped with the EFM32 microcontroller.
+It is specifically designed for low-power applications, having energy-saving
+peripherals, different energy modes and short wake-up times.
+
+The starter kit is equipped with an Advanced Energy Monitor. This allows you to
+actively measure the power consumption of your hardware and code, in real-time.
```

Comment:
I wouldn't have expected that from this cheap board

Reply:
Whoops, that's from a Silicon labs board, I will remove that!

Apart from that, you should be able to use a Silicon Labs board for power measurements ;-)